content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -13,12 +13,7 @@ algolia:
   tags: ['otel instrumentation']
 ---
 
-[Instrumentation][1] is the process of adding code to your application to capture and report observability data.
-[Automatic instrumentation][2] is a way to instrument applications and libraries without modifying their source code.
-Both OpenTelemetry and Datadog provide automatic instrumentation in their SDKs.
-
-Datadog SDKs support adding [OpenTelemetry instrumentation libraries][3] to their existing automatic instrumentation.
-This provides observability for libraries not covered by Datadog SDKs without changing SDKs.
+Datadog’s support for OpenTelemetry instrumentations now allows you to leverage community-built OpenTelemetry-compatible instrumentations. You can now extend Datadog’s tracing capabilities to these frameworks, ensuring no part of your tech stack is left unmonitored.
```

Comment:
From the automated suggestions:
- Use straight quotes instead of smart quotes.
- Avoid temporal words like 'now'.

```suggestion
Datadog's support for OpenTelemetry instrumentations let you to leverage community-built OpenTelemetry-compatible instrumentations. You can extend Datadog's tracing capabilities to these frameworks, ensuring no part of your tech stack is left unmonitored.
```

Reply:
Addressed